pkg/gor/debian/rules

```diff
@@ -2,6 +2,7 @@
 
 export GOPATH=$(CURDIR)
 export GOBIN=$(CURDIR)/debian/gor/usr/bin
+export VERSION="0.10.1"
```

Comment:
I was hoping there was a way to get this, not duplicating VERSION in here. Thanks @alext